src/commands/mobile_app/upload.rs

```diff
@@ -0,0 +1,26 @@
+use anyhow::Result;
+use clap::ArgAction;
+use clap::{Arg, ArgMatches, Command};
+
+use crate::utils::args::ArgExt;
+
+pub fn make_command(command: Command) -> Command {
+    command
+        .about("Upload mobile app files to a project.")
```

Comment:
Please also mark this as experimental; the parent command's about text, which already includes an experimental tag, does not show up when you run `sentry-cli mobile-app upload --help`, so we should include it here as well.

```suggestion
        .about("[EXPERIMENTAL] Upload mobile app files to a project.")
```